src/governance.ts

```diff
@@ -208,6 +213,50 @@ export class GovernanceCanister {
     return proposalInfo ? toProposalInfo(proposalInfo) : undefined;
   };
 
+  /**
+   *
+   * Registers vote for a proposal from the neuron passed
+   *
+   * @param neuronId: NeuronID
+   * @param vote: Vote
+   * @param ProposalId: ProposalId
+   * @returns EmtpyResponse
+   */
+  public registerVote = async ({
+    neuronId,
+    vote,
+    proposalId,
+  }: {
+    neuronId: NeuronId;
+    vote: Vote;
+    proposalId: ProposalId;
+  }): Promise<EmptyResponse> => {
+    const request = toRegisterVoteRequest({ neuronId, vote, proposalId });
+    const rawResponse = await this.certifiedService.manage_neuron(request);
```

Comment:
maybe destructable? 

```
const {command} = await this.certifi...
const response  = command[0];
```

note: can live without explicit types but not against 😉

Reply:
not a strong opinion on my side, I can change it. This is the reason I would like to have eslint. To avoid that you need to check for these kind of things.